docs/source/template/task.rst

```diff
@@ -0,0 +1,66 @@
+.. _contributing_task:
+
+********
+The Task
+********
+
+Once you've implemented a Flash :class:`~flash.core.data.data_module.DataModule` and some backbones, you should implement your :class:`~flash.core.model.Task` in `model.py <https://github.com/PyTorchLightning/lightning-flash/blob/master/flash/template/classification/model.py>`_.
+The :class:`~flash.core.model.Task` is responsible for: setting up the backbone, performing the forward pass of the model, and calculating the loss and any metrics.
+Remember that, under the hood, the Flash :class:`~flash.core.model.Task` is simply a :any:`pytorch_lightning:lightning_module` with some helpful defaults.
+
+Task
+^^^^
+You can override the base :class:`~flash.core.model.Task` or any of the existing :class:`~flash.core.model.Task` implementations.
+For example, in our scikit-learn example, we can just override :class:`~flash.core.classification.ClassificationTask` which provides good defaults for classification.
+
+You should attach your backbones registry as a class attribute like this:
+
+.. code-block:: python
+
+    backbones: FlashRegistry = TEMPLATE_BACKBONES
+
+In the :meth:`~flash.core.model.Task.__init__`, you will need to configure defaults for the:
```

Comment:
Maybe add a title:

Model Architecture and parameters
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^